src/hooks/useTxBatchTrades.tsx

```diff
@@ -37,53 +38,49 @@ export function useTxBatchTrades(
   const [error, setError] = useState('')
   const [txBatchTrades, setTxBatchTrades] = useState<TxBatchTrades>({ trades: [], transfers: [] })
   const [accounts, setAccounts] = useState<Accounts>()
+  const txOrders = usePrevious(JSON.stringify(orders?.map((o) => ({ owner: o.owner, kind: o.kind }))))
```

Comment:
we should write some comment here to explain this tricky solution. Is not that obvious why the usePrevious hook and the stringifying are needed